GTG/gtk/editor/serialize.py

```diff
@@ -230,7 +230,7 @@ def parsexml(self, buf, ite, element):
                         s = buf.get_iter_at_mark(start)
                         e = buf.get_iter_at_mark(end)
                         if n.nodeName == "link":
-                            anchor = n.getAttribute("target")
+                            anchor = n.get("target")
```

Comment:
My entire life was changed by this, by the way. I always did `elem.attrib.get('attrName')`; I had no idea I could just do `elem.get('attrName`)`. So thanks. ;)

Reply:
Glad to hear! :D